specification/subscription/resource-manager/Microsoft.Subscription/preview/2024-08-01-preview/subscriptions.json

```diff
@@ -0,0 +1,1396 @@
+            "$ref": "./examples/getTenantPolicyList.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Subscription tenant policy details for user's tenant.",
+            "schema": {
+              "$ref": "#/definitions/GetTenantPolicyListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.Subscription/policies/default": {
+      "get": {
+        "description": "Get Billing Account Policy.",
+        "operationId": "BillingAccount_GetPolicy",
+        "x-ms-examples": {
+          "GetBillingAccountPolicy": {
+            "$ref": "./examples/getBillingAccountPolicy.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/BillingAccountPoliciesResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/billingAccountIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/changeTenantRequest/default": {
+      "put": {
+        "operationId": "Subscriptions_PutTargetDirectory",
+        "x-ms-examples": {
+          "putTargetDirectory": {
+            "$ref": "./examples/putTargetDirectory.json"
+          }
+        },
+        "description": "The operation to initiate Subscription Changed Request",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/targetDirectoryRequestParameters"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TargetDirectoryResult"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
```

Comment:
Use `ErrorResponse` from common types? I saw you made the change in some places. Can you change them all?

Reply:
sure @gary-x-li